dmoj/contrib/peg.py

```diff
@@ -7,11 +7,11 @@
 
 class ContribModule(DefaultContribModule):
     name = 'peg'
-    repartial = re.compile(r'^(\d+)\n(\d+)$', re.M)
+    repartial = re.compile(r'^(\d+\.?\d*)\n(\d+\.?\d*)$', re.M)
```

Comment:
This doesn't really match all floats, e.g. ones using scientific notation. Might be best to just split the lines and `float(...)`.

Reply:
Changed to checking for 2 consecutive lines that have valid floats.